docs/reference/migration/migrate_6_0/rest.asciidoc

```diff
@@ -19,6 +19,9 @@ In previous versions of Elasticsearch, having a proper Content-Type for the data
 Elasticsearch 6.0.0 enforces that all requests with a body must have a supported Content-Type and this type will
 be used when parsing the data.
 
+When using the `source` query string parameter, the `source_content_type` parameter must also be specified with
+the media type of the source.
+
```

Comment:
shall we add a bullet point on the removal of all those setters from the Java api? If we don't want to list them all, shall we just explain the change which type of methods it affects and how to fix the compile error?